### PR TITLE
OCPBUGS-30084: Add infrastructure annotations

### PR DESCRIPTION
**Pull request by bertinatto**
CC @openshift/storage

### CODE REVIEW

File: config/manifests/preview/gcp-filestore-csi-driver-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -14,6 +14,14 @@ metadata:
     createdAt: "2021-07-14T00:00:00Z"
     description: Install and configure GCP Filestore CSI driver.
     olm.skipRange: ">=4.11.0-0 <4.12.0"
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/proxy-aware: "true"
+    features.operators.openshift.io/tls-profiles: "true"
```
Comment (jsafrane) [inline config/manifests/preview/gcp-filestore-csi-driver-operator.clusterserviceversion.yaml:20]: I think we added TLS config support in 4.15.

Comment (bertinatto) [reply config/manifests/preview/gcp-filestore-csi-driver-operator.clusterserviceversion.yaml]: Fixed

**Pull request merged**